### ISSUE TITLE
Switch between decks on the main Decks screen

### ISSUE THREAD

**Issue by chkiss** [Liresol/anki-custom-shortcuts#76]
Is there any way to switch between decks on the main screen? I can't seem to find it. I would love to be able to stick to the keyboard during my whole study session, but currently I have to resort to the mouse to select which deck I want to study.

Thanks for a great add-on!

**Comment by Liresol** [comment]
Hello,
Sorry for being a bit late, but hopefully this can still help.
For deck navigation, you can use `/` (`m_toolbox study`) and use the keyboard to scroll through and select a deck that way.
If needed, you can also switch back to the deck browser (the starting menu) from studying with `d` (`main deckbrowser`) and switch back to studying the current deck with `s` (`main study`).

**Comment by chkiss** [comment]
No problem on the delay!

I should have mentioned this, but I stick to a numpad for all of my other navigation. This does work if I remove numlock and use the arrow keys to get to the right deck, then switch back to numlock for my custom `s` shortcut. However, the `/` screen doesn't show review counts etc, so I was hoping to do such navigation from the main "Decks" screen.

I guess this may not be possible since Anki doesn't support arrow keys (or similar) to switch between decks natively. Thank you for responding, though.

**Comment by Liresol** [comment]
Yeah, it kind of sucks that among the infinite number of keyboard shortcuts in Anki, deck navigation in the deck browser isn't among them as far as I can tell.

A non-arrow key workaround I can think of is to switch back to the deck browser with `d` to look at the main info, then use `/` and use the filter to fill in the deck you want. It's still not that great, but it's probably the best we can do with the limited navigation tool set.
